packages/components/src/internal/components/administration/BasePermissions.tsx

```diff
@@ -114,7 +116,8 @@ export const BasePermissionsImpl: FC<BasePermissionsImplProps> = memo(props => {
 
         return (
             <>
-                <CreatedModified row={row} />
+                {lkVersion?.()}
+                {!lkVersion && <CreatedModified row={row} />}
```

Comment:
Seems odd that we would show this `CreatedModified` data only when not showing the version, but I guess the check for `lkVersion` is really a surrogate for an explicit "showVersion" property, and is passed in only from the Settings page.